release-notes/major9/09.23.0/valtimo-frontend-libraries.md

```diff
@@ -39,8 +39,11 @@ The following features were added:
     ]
   }
   ```
-  
 
+* **Added warning message to user management page**
+
+  On the user management page an admin can now see a warning message that notifies them about the fact that user management is handled separately in Keycloak in case Keycloak is used for IAM. The message can be customized by using the translation key `User management not supported`.
```

Comment:
Adjust translation key, see comment other PR. Also, take into account line endings, this sentence is too long

Reply:
done